### PR TITLE
Add bear-fied badge asset (generated with shields.io)

### PR DESCRIPTION
**Pull request by posita**
Includes re-fetching `update.bash` script because you just never know. :shrug:
Step 1 of _n_ pursuant to beartype/beartype#44 (where _n_ is hopefully equal to 2).

### CODE REVIEW

**Comment by leycec** [comment]
**ZOMG.** However much DropBox is paying you, <sup>...*it's alot isn't it*</sup> I wish Humanity™ would pay you double that so that you could work on behalf of our entire species and help make this sad pale blue dot that much saner, healthier, and happier. Like, you're just amazing!

And *soooo* usefully funny. It's hard to be useful; it's even harder to be funny; but usefully integrating the funny with the worky? That's basically infeasible. Yet you do these infeasible things and then you *keep doing them*. Like, that updater script:

```
# Of course curl is everywhere, right?
if type >/dev/null 2>&1 curl ; then
    exec curl >"${LOC}" "${BADGE_URL}"
# Hah! You thought we'd give up so easily? Not a chance! Surely wget is here.
elif type >/dev/null 2>&1 wget ; then
    exec wget --output-document "${LOC}" "${BADGE_URL}"
# Oh, so you want to play hardball, eh? Batteries included.
elif type >/dev/null 2>&1 "${PYTHON:-python3}" ; then
    exec "${PYTHON:-python3}" -c "${_FETCH_PY}" "${LOC}" "${BADGE_URL}"
# Goonies never say "die"!
elif type >/dev/null 2>&1 python ; then
    exec python -c "${_FETCH_PY}" "${LOC}" "${BADGE_URL}"
# You have bested us, gentleperson. Well played. Are you trying to run this on a router
# or a washing machine or something?
```

Yes. Yes, we are. It is known by some that HP-UX shells accessible only over washing machine serial port connections make the safest, most accurate, and industrial-strength testbeds for GLWTPL-licensed scripts.

**So.** @posita, if that is indeed your real username. You go above and beyond the call of free-as-in-beer duty by supporting all possible URL fetchers including those that really shouldn't even exist that require you to manually hack out a 7-line Bourne Shell `HEREDOC` statement that you then surreptitiously feed into Python, <sup>...*as if we wouldn't notice that*</sup> and then you one-up yourself with obscure 80's references ala "I love the dark but I hate nature!" that only you and I and the lone chatbot over there grok, and then you one-up even that by licensing all of this extremely well-tested robustness under the [funniest software license that Man and/or Beast have yet devised](https://github.com/me-shaon/GLWTPL), which I now quote for completeness:

> You wrote some code but not quite proud of it...

> "Whatever done is done"

```
When I wrote this, only God and I understood what I was doing.
Now, only God knows.
```

```
The author has absolutely no fucking clue what the code in this project
does. It might just fucking work or not, there is no third option.

                GOOD LUCK WITH THAT SHIT PUBLIC LICENSE
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION, AND MODIFICATION

  0. You just DO WHATEVER THE FUCK YOU WANT TO as long as you NEVER LEAVE
A FUCKING TRACE TO TRACK THE AUTHOR of the original product to blame for
or held responsible.
```

In short, take all my GitHub dharma. Take all of it! :wheel_of_dharma: :relieved: :wheel_of_dharma:

**Comment by posita** [comment]
I'm glad it … _landed_. 😜 It is, of course, these interactions that keep me working on this and not more ~mundane~…erm…_important_ things. Maybe I should just quit this day job and work for [Umbrella Corporation DBA] Humanity™ full time…. 🤔💭

As my father is fond of quoting, "We are not here for a long time. We are here for a good time." This is a good time. (How often do we get to say that about collaborative software development of runtime type checkers? Thank you for that. 😊)

**Comment by leycec** [comment]
> [Umbrella Corporation DBA] Humanity™

*uhoh*

This ain't gunna end well, Milla Jovovich! Cue saccharine-sweet little girl voiceover dripping with surely unintentional malice:

`"Welcome to Raccoon City, Mr. Bogosian. We hope you enjoy your time as a still-human DBA at [Umbrella Corporation], where the reputable career opportunities in cutting-edge ~~illicit bioweapons manufacturing~~ FDA-approved genetic engineering are matched only by our thrilling late-night exercise routine at the Spenser Mansion."`

> As my father is fond of quoting, "We are not here for a long time. We are here for a good time."

...your Dad is a luminary of truth and probable Zen master. I *knew* it. High-five that archetypal paragon of wisdom for @leycec, for Father Bogosian plays for Team Good Guys. :hand: 

I'm delighted I could make stay here that much gooder. Thanks again for the tremendous goodwill you throw at everything, @posita. You are the sum of all awesomeness.
